### PR TITLE
static template

### PR DESCRIPTION
**Pull request by Vsumskiy**

### CODE REVIEW

File: src/app/pages/favorites/favorites-page/favorites-page.component.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<app-favorite-card></app-favorite-card>
```
Comment (iyarymovych-reenbit) [inline src/app/pages/favorites/favorites-page/favorites-page.component.html:1]: I see `app-joke-card` component. Why do you need to create another joke components for favorite jokes?
I would suggest to have one reusable joke-card component that has styles configuration inputs.

File: src/app/pages/favorites/favorites-page/favorites-page.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+hello
```
Comment (iyarymovych-reenbit) [inline src/app/pages/favorites/favorites-page/favorites-page.component.html:3]: Why do you need this 'hello'?

File: src/app/pages/home/joke-card/joke-card.component.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<div class="joke-card">
+  <div class="card-content">
+    <img src="../assets/images/heart.svg" alt="" class="heart" />
+    <div class="comment">
+      <img src="../assets/images/comment.svg" alt="" />
+    </div>
+    <div class="card-content-inner">
+      <span
```
Comment (iyarymovych-reenbit) [inline src/app/pages/home/joke-card/joke-card.component.html:8]: `<span`
Keep tags staff on the same line:
`<span>`

File: src/app/pages/favorites/favorites-page/favorites-page.component.html (hunk 1 above)
Comment (Vsumskiy) [reply src/app/pages/favorites/favorites-page/favorites-page.component.html:1]: Done.

File: src/app/pages/favorites/favorites-page/favorites-page.component.html (hunk 2 above)
Comment (Vsumskiy) [reply src/app/pages/favorites/favorites-page/favorites-page.component.html:3]: Removed.

File: src/app/pages/home/joke-card/joke-card.component.html (hunk 3 above)
Comment (Vsumskiy) [reply src/app/pages/home/joke-card/joke-card.component.html:8]: Got it!

**Pull request merged**